src/main/java/org/jmxtrans/agent/Query.java

```diff
@@ -151,7 +151,7 @@ private void collectAndExportAttribute(MBeanServer mbeanServer, OutputWriter out
             try {
                 attributeValue = mbeanServer.getAttribute(objectName, attribute);
             } catch (Exception ex) {
-                logger.warning("Failed to fetch attribute for '" + objectName + "'#" + attribute + ", exception: " + ex.getMessage());
+                logger.info("Failed to fetch attribute for '" + objectName + "'#" + attribute + ", exception: " + ex.getMessage());
```

Comment:
I had added this warning in #34. But I realized that this seems like a common occurance; hence I thought of dumbing it down a bit. Let me know what do you want to do here.